atom/common/platform_util_mac.mm

```diff
@@ -128,7 +129,17 @@ bool OpenItem(const base::FilePath& full_path) {
   return status == noErr;
 }
 
-bool OpenExternal(const GURL& url, bool activate) {
+bool openURLInWorkspace(NSURL* ns_url, NSUInteger launchOptions) {
```

Comment:
We use AllCamelCase for function names, so `openURLInWorkspace` => `OpenURLInWorkspace`, `openExternal` => `OpenExternal`.

And helper functions should be put into anonymous namespace, you can find an example in `platform_util_linux.cc`.